packages/twenty-server/src/engine/metadata-modules/object-metadata/services/object-metadata-migration.service.ts

```diff
@@ -264,4 +267,44 @@ export class ObjectMetadataMigrationService {
       ],
     );
   }
+
+  public async recomputeEnumNames(
+    existingObjectMetadata: ObjectMetadataEntity,
+    updatedObjectMetadata: ObjectMetadataEntity,
+    workspaceId: string,
+  ) {
+    const fieldMetadataToUpdate = await this.fieldMetadataRepository.find({
+      where: {
+        objectMetadataId: updatedObjectMetadata.id,
+        workspaceId,
+        type: In([
```

Comment:
is there a way for us not to forget to update this list if we introduce a new enum?

Reply:
We have an isEnumFieldMetadataType but you can't call it in a request like this, we would like to extract the list of the possible types indeed. However this query will fetch composite fields as well so we need to take this into account. Most of the code won't expect ACTOR for example because fields are decomposed and I feel like here would be the only exception 🤔